test/public/runs/detail.test.js

```diff
@@ -464,4 +466,26 @@ module.exports = () => {
         expect(await page.$eval('.external-links a:nth-of-type(2)', ({ href }) => href))
             .to.equal('http://localhost:8082/?page=layoutShow&runNumber=108&definition=COMMISSIONING&pdpBeamType=cosmic&runType=PHYSICS');
     });
+
+    it('should display links to environment in ECS if run is running', async () => {
+        // Test for not running run
+        await goToPage(page, 'run-detail', { queryParameters: { runNumber: 104 } });
+        await page.waitForSelector('.external-links a:nth-of-type(3)', { hidden: true, timeout: 250 });
+
+        // Create running run
+        await runService.create({ runNumber: 1010, timeTrgStart: new Date(), environmentId: 'CmCvjNbg' });
+        await goToPage(page, 'run-detail', { queryParameters: { runNumber: 1010 } });
```

Comment:
No more goToPage except for new test suite. 
Here I see no point in not creating the run at the start of the test, or even better at the start of the test suite.
(I saw pupeteer issues when calling services directly in between tests, no idea why)

Reply:
refactored